### PR TITLE
Added WSL instructions txt

### PR DESCRIPTION
**Pull request by ffermo**
Added instructions txt file under /docs/ and added a line in the README to hyperlink to txt file.

### CODE REVIEW

**Comment by ffermo** [comment]
@tgsachse for review.

File: docs/WINDOWSINSTALL.txt [hunk 1]
```diff
@@ -0,0 +1,80 @@
+This is working for Windows 10 version 1909. You can verify which Windows version you are on by using keyboard shortcut WIN + r and typing 'winver'.
```
Comment (tigressine) [inline docs/WINDOWSINSTALL.txt]: Would rather this was part of the readme where the other installation instructions are.

- Could you add a new section after Customized Installation, perhaps called WSL Configuration or something
- In this section, include all of the parts of 1)
- Next, create a step where you tell users to follow the instructions in Typical Installation (we don't want to repeat instructions here and in typical installation for installing ROS/EZ-RASSOR repository, and we also don't want to include the instructions to install ROS at all bc they are the most up to date on the ros wiki, no point in maintaining them on our side (and possibly having out of date info)).
- Add section 6)
- Drop the test instructions (testing is covered in the readme already)

**Review by tigressine — CHANGES_REQUESTED** [review]
Nice instructions! Had one ask regarding organization, thanks for doing this :)

File: docs/WINDOWSINSTALL.txt (hunk 1 above)
Comment (ffermo) [reply docs/WINDOWSINSTALL.txt]: Changes made! Feel free to modify or request any other changes.

**Review by tigressine — APPROVED** [review]
Great work and nice attention to detail.

**Pull request merged**